### PR TITLE
Add information about local template directories, reindent, fix some …

### PR DESCRIPTION
**Pull request by kozross**
Relates to #6.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,58 +2,75 @@
...
- * `<repository>` — Git URL of a template repository (for GitHub you can just use `organization/repo`),
- * `<directory>` — where to create a project.
 
-You will be asked to choose a subset of template features you want to include, and to provide some variable values, and then the project will be created.
+* `<repository>` — Git URL of a template repository (for GitHub you can just
+    use `organization/repo`).
+* `<directory>` — where to create a project.
+
+You can also pass a local path (either to a clone of an online repo, or just a
+regular folder) instead of a Git URL.
```
Comment (rtimush) [inline README.md]: Let's mention that a clone must be made with a `--mirror` flag. Also, a regular folder that is not a git repository will not work, the template must be a git repo.

File: README.md [hunk 2]
```diff
@@ -2,58 +2,75 @@
...
 
 ### macOS
 
 You can install `tenpureto` with [Homebrew](https://brew.sh):
 
-```sh
-$ brew tap rtimush/tap # you need it only once
-$ brew install tenpureto
+```sh 
+$ brew tap rtimush/tap # you need it only once $ brew install tenpureto
```
[the remark this answers is not in the document]
Comment (kozross) [reply README.md]: Sorry, my bad - fixed in last commit.

File: README.md (hunk 1 above)
Comment (kozross) [reply README.md]: Changed the wording - is that better?

Comment (rtimush) [reply README.md]: Yep, now sounds correct

**Pull request merged**